content/docs/command-reference/unfreeze.md

```diff
@@ -35,56 +38,38 @@ be unfrozen. Use `dvc update` on them to update the file, directory, or
 
 ## Examples
 
-First, let's create a simple DVC-file:
+First, let's create a dummy stage that copies `foo` to `bar`:
 
 ```dvc
 $ echo foo > foo
 $ dvc add foo
-Saving information ...
-
-$ dvc run -d foo -o bar cp foo bar
-Running command:
-  cp foo bar
-...
+$ dvc run -n make_copy -d foo -o bar cp foo bar
 ```
 
-Then, let's change the file `foo` that the stage described in `bar.dvc` depends
-on:
-
-```dvc
-$ rm foo
-$ echo foo1 > foo
-$ dvc status
-
-bar.dvc
-        deps
-                changed:  foo
-foo.dvc
-        outs
-                changed:  foo
-```
+> See `dvc run` for more details.
 
-Now, let's freeze the `bar` stage:
+Then, let's change the file `foo` that the stage `make_copy` depends on, and
+freeze stage as well, to see what's the project status after that:
 
 ```dvc
-$ dvc freeze bar.dvc
+$ dvc freeze make_copy
 $ dvc status
```

Comment:
We mention change of `foo` but we don't do that

Reply:
Fixed!